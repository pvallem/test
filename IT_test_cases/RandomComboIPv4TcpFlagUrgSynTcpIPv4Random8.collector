bidir_flow_info {
  key {
    src_address: "\001\002\003\004"
    dst_address: "\005\006\007\010"
    src_port: 111
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000000
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      accumulated_flags {
        syn: 1
        urg: 1
      }
      sequence_num: 20
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      urg_ptr_not_consistent: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 3
    }
    end_time: 1388563200000040
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "z|\334\335"
    dst_address: "\2737\236\267"
    src_port: 6971
    dst_port: 4848
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000010
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000010
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\275\323x\017"
    dst_address: "\311v\271\203"
    src_port: 56645
    dst_port: 36735
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200000030
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000030
    sensor_id: "127.0.0.1:5675"
  }
}
