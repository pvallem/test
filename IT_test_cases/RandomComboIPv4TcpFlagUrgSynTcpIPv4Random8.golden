flow_info {
  key {
    src_address: "\001\002\003\004"
    dst_address: "\005\006\007\010"
    src_port: 111
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  start_time: 1388563200000000
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    accumulated_flags {
      syn: 1
      urg: 1
    }
    sequence_num: 20
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    urg_ptr_not_consistent: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 3
  }
  end_time: 1388563200000040
}
flow_info {
  key {
    src_address: "z|\334\335"
    dst_address: "\2737\236\267"
    src_port: 6971
    dst_port: 4848
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV4
  }
  start_time: 1388563200000010
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000010
}
flow_info {
  key {
    src_address: "\311v\271\203"
    dst_address: "\275\323x\017"
    src_port: 36735
    dst_port: 56645
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV4
  }
  start_time: 1388563200000030
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000030
}
